apps/src/templates/progress/ProgressLevelSet.jsx

```diff
@@ -1,43 +1,16 @@
 import React, { PropTypes } from 'react';
 import Radium from 'radium';
-import FontAwesome from '../FontAwesome';
 import ProgressBubbleSet from './ProgressBubbleSet';
 import color from '@cdo/apps/util/color';
 import i18n from '@cdo/locale';
 import { levelType } from './progressTypes';
-
-import { BUBBLE_COLORS } from '@cdo/apps/code-studio/components/progress/ProgressDot';
+import { getIconForLevel } from './progressHelpers';
+import ProgressPill from './ProgressPill';
 
 const styles = {
   table: {
     marginTop: 12
   },
-  levelPill: {
-    // TODO - fixed width isn't great for i18n. likely want to come up with some
-    // way of having this be dynamic, but the same size across all instances
-    width: 110,
-    textAlign: 'center',
-    borderWidth: 1,
-    borderStyle: 'solid',
-    borderColor: color.lighter_gray,
-    display: 'inline-block',
-    fontFamily: '"Gotham 5r", sans-serif',
-    borderRadius: 20,
-    paddingLeft: 20,
-    paddingRight: 20,
-    paddingTop: 5,
-    paddingBottom: 5
-  },
-  hoverStyle: {
-    ':hover': {
-      textDecoration: 'none',
-      color: color.white,
-      backgroundColor: color.level_current
-    }
-  },
-  buttonText: {
-    marginLeft: 10
-  },
```

Comment:
This got moved into a ProgressPill component so that we could also use it for unplugged bubbles